website/static/js/filepage/revisions.js

```diff
@@ -44,6 +45,7 @@ var FileRevisionsTable = {
                 url: self.file.urls.revisions,
                 beforeSend: $osf.setXHRAuthorization
             }).done(function(response) {
+                response = fangorn.DefaultOptions.lazyLoadPreprocess.call(this, response);
```

Comment:
Can you get `lazyLoadPreprocess` from the treebeard object instead of the global defaults?